<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vineger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="browse_butt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="code_butt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decode_butt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_butt">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="display_butt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
